dune_client/interface.py

```diff
@@ -19,3 +20,24 @@ def refresh(self, query: Query) -> ResultsResponse:
         Executes a Dune query, waits till query execution completes,
         fetches and returns the results.
         """
+
+    @abc.abstractmethod
+    def refresh_csv(self, query: Query) -> ExecutionResultCSV:
+        """
+        Executes a Dune query, waits till execution completes,
+        fetches and the results in CSV format
+        (use it load the data directly in pandas.from_csv() or similar frameworks)
+
+        this Dune API only returns the raw data in CSV format, it is faster & lighterweight
+        use this method for large results where you want lower CPU and memory overhead
+        """
+
+    @abc.abstractmethod
+    def refresh_into_dataframe(self, query: Query) -> Any:
```

Comment:
I believe you can use the Dataframe type here... although I guess it implies another `from pandas import DataFrame` and becomes more difficult with the unsupported dependency. I guess we have to leave it as Any.